Gruntfile.js

```diff
@@ -146,10 +154,8 @@ module.exports = function(grunt) {
             executableName: 'wire-desktop',
             afterInstall: 'bin/deb/after-install.tpl',
             afterRemove: 'bin/deb/after-remove.tpl',
-            desktop: {
-              "StartupWMClass": "Wire"
-            },
-            category: 'Network',
+            desktop: DESKTOP_SPEC,
+            category: 'Network;InstantMessaging;Chat;VideoConference',
```

Comment:
maybe we could put that in a variable as well and rename the other one to something like: `LINUX_DESKTOP` and `LINUX_CATEGORY`.

WDYT?

Reply:
Is the category here used anywhere other than the `.desktop` file? It works fine if I just nest the categories in the desktop spec.